js-extensions/EXTENSIONS.md

```diff
@@ -12,8 +12,8 @@ JSe `@jenkins-cd/js-extensions` module exports 2 things:
 
 The `ExtensionStore` API is very simple, all public methods are asynchronous:
 
-- `getExtensions(extensionPointName, [type,] onload)`
-    This method will async load data and type information as needed, and call the onload handler with a list of extension exports, e.g. the React classes or otherwise exported references.
+- `getExtensions(extensionPointName, [filter,] onload)`
+    This method will async load data and filter the information as needed, and call the onload handler with a list of extension exports, e.g. the React classes or otherwise exported references.
```

Comment:
I'm imagining what this might mean here, but I think a concrete use case example would help. "filter as needed" is a bit ambiguous.